Game2048Intelligence/src/main/java/models/HeuristicScore.java

```diff
@@ -0,0 +1,13 @@
+package models;
+
+public class HeuristicScore {
```

Comment:
This would be a good implementation of the immutables library if we tagged it with @Immutable!

Reply:
What